packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -79,58 +68,29 @@ class MouseTrackerAnnotation {
   /// exited the annotated region when the annotated region still exists.
   ///
   /// This callback is triggered when the pointer has stopped to be contained
-  /// by the region, except when it's caused by the removal of the render object
-  /// that owns the annotation. More specifically, the callback is triggered by
-  /// the following cases:
-  ///
-  ///  * An annotated region that used to contain a pointer has moved away.
-  ///  * A pointer that used to be within an annotated region has been removed.
-  ///  * A pointer that used to be within an annotated region has moved away.
-  ///
-  /// And is __not__ triggered by the following case,
-  ///
-  ///  * An annotated region that used to contain a pointer has disappeared.
-  ///
-  /// The last case is the only case when [onExit] does not match an earlier
-  /// [onEnter].
-  /// {@template flutter.mouseTracker.onExit}
-  /// This design is because the last case is very likely to be
-  /// handled improperly and cause exceptions (such as calling `setState` of the
-  /// disposed widget). There are a few ways to mitigate this limit:
-  ///
-  ///  * If the state of hovering is contained within a widget that
-  ///    unconditionally attaches the annotation (as long as a mouse is
-  ///    connected), then this will not be a concern, since when the annotation
-  ///    is disposed the state is no longer used.
-  ///  * If you're accessible to the condition that controls whether the
-  ///    annotation is attached, then you can call the callback when that
-  ///    condition goes from true to false.
-  ///  * In the cases where the solutions above won't work, you can always
-  ///    override [State.dispose] or [RenderObject.detach].
-  /// {@endtemplate}
-  ///
-  /// Technically, whether [onExit] will be called is controlled by
-  /// [MouseTracker.attachAnnotation] and [MouseTracker.detachAnnotation].
+  /// by the region for any reason, which means it always matches an earlier
+  /// [onEnter]. This is different from [MouseRegion.onExit], which is not
+  /// triggered in certain cases and does not always match `onEnter`.
```

Comment:
onEnter -> [MouseRegion.onEnter] to make it clear what onEnter is meant?

Reply:
Yes, fixed.